rviz_common/src/rviz_common/viewport_mouse_event.cpp

```diff
@@ -57,9 +57,14 @@ ViewportMouseEvent::ViewportMouseEvent(RenderPanel * p, QWheelEvent * e, int lx,
 : panel(p),
   type(e->type()),
   device_pixel_ratio(static_cast<int>(panel->getRenderWindow()->devicePixelRatio())),
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+  x(e->position().x() * device_pixel_ratio),
+  y(e->position().y() * device_pixel_ratio),
+#else
   x(e->x() * device_pixel_ratio),
   y(e->y() * device_pixel_ratio),
-  wheel_delta(e->delta()),
+#endif
+  wheel_delta(e->angleDelta().x()),
```

Comment:
Looks like this line broke zooming with the mouse wheel (https://github.com/ros2/rviz/issues/605).

Reply:
Fix in https://github.com/ros2/rviz/pull/623